packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -17,6 +17,34 @@ import type {
 
 export type AuthorsMap = {[authorKey: string]: Author};
 
+interface Socials {
+  twitter: string;
+  github: string;
+}
+
+const normalizeSocials = (value: Socials) => {
+  const socialPlatforms = {
+    twitter: 'https://twitter.com/',
+    github: 'https://github.com/',
+    linkedin: 'https://www.linkedin.com/in/',
+    stackoverflow: 'https://stackoverflow.com/users/',
+  };
+
+  (Object.keys(socialPlatforms) as (keyof Socials)[]).forEach((platform) => {
+    if (
+      value[platform] &&
+      !value[platform]!.startsWith(socialPlatforms[platform])
```

Comment:
I'd say that we only add the prefix url if the current value does not start with HTTP or https?

We should allow someone using a tracked link shortener to their profile if they want to for example (think `https://bit.ly/sebastienlorber-twitter` should be ok). 

need unit tests to cover this